assets/src/application/ui/display/loadingIndicator/LoadingIndicator.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { LoadingOutlined } from '@ant-design/icons';
+import { Spin } from 'antd';
+import { SpinProps } from 'antd/lib/spin';
+
+const LoadingIndicator: React.FC<SpinProps> = (props) => {
+	const antIcon = <LoadingOutlined style={{ fontSize: 24 }} spin />;
```

Comment:
We should avoid inline styling or at least avoid using object literals in JSX attributes. It creates unnecessary rerenders.

Reply:
it was based on antd example:
![image](https://user-images.githubusercontent.com/1477453/76301845-a2595980-62c7-11ea-8a3b-9e6cc7de6b87.png)

Reply:
but I do agree with your comment, thx, suggestion applied